src/bbs2gh/Commands/MigrateRepoCommand.cs

```diff
@@ -44,98 +38,92 @@ public class MigrateRepoCommand : Command
             {
                 IsRequired = true
             };
-            var ssh = new Option("--ssh")
-            {
-                IsRequired = false,
-                IsHidden = true,
-                Description = "Uses SSH protocol instead of HTTPS to push a Git repository into the target repository on GitHub."
-            };
-            var wait = new Option("--wait")
+            var githubPat = new Option<string>("--github-pat")
             {
-                IsRequired = false,
-                Description = "Synchronously waits for the repo migration to finish."
+                IsRequired = false
             };
-            var adoPat = new Option<string>("--ado-pat")
+            var githubApiUrl = new Option<string>("--github-api-url")
```

Comment:
I don't think we should include this arg. This is primarily to support GHAE, which I don't want to do right now (hopefully never).

Reply:
After talking about this in a 🍐ing session, we decided to take this option out and include it as an environment variable in the future.  This makes it easier to test in development, but for customers, it doesn't really serve a very useful feature, and adds code complexity 👍

Reply:
Removed in 273316e0a520cabc669fe582d56ea306a27d3cf4!